specs/latest/1.0/index.html

```diff
@@ -3481,6 +3481,14 @@ <h2 class="no-toc">Table of contents</h2>
 
 </p>
 
+    <h3>No Shader Binaries</h3>
+
+<p>
+
+Accessing binary representations of compiled shaders is not supported in the WebGL API. This includes OpenGL ES 2.0 ShaderBinary entry point. In addition, querying shader binary formats and shader compiler with getParameter is not supported in the WebGL API.
```

Comment:
Thanks for your review, @kenrussell . I have updated this pull request accordingly, please take another look.